src/test/resources/config/test-it-config.yaml

```diff
@@ -26,6 +26,9 @@ goCardless:
   webhookSecret: secret
   environment: sandbox
 
+encryption:
+  encryptDBSalt: $2a$10$IhaXo6LIBhKIWOiGpbtPOu
```

Comment:
I think each record has a salt and is saved in the database together with the encrypted value. The fact that public auth is using a salt is ok (since is not being used to encrypt user input)

Reply:
done, will be up in the next pr